libs/ha/ha_common_libs_k8s.py

```diff
@@ -1777,15 +1784,19 @@ def iam_bucket_cruds(self, event, s3_obj=None, user_crud=False, num_users=None,
                 try:
                     LOGGER.debug("Creating %s user", i_i)
                     user = None
-                    user = self.mgnt_ops.create_account_users(nusers=1)
+                    if not header:
+                        user = self.mgnt_ops.create_account_users(nusers=1)
+                    else:
+                        user = self.create_iam_user_with_header(header=header)
```

Comment:
can't we use this method only and just pass header as header or none?

Reply:
When we want to create user without header, we need to login and login is done as part of decorator over create_account_users(). We are not using that decorator in create_iam_user_with_header().

Reply:
> can't we use this method only and just pass header as header or none?

But then, in already written code of different test module, we have to update test to get the header as using create_account_users() we are not passing any header value taken before.